### PR TITLE
Add secret to opsrc spec

### PR DESCRIPTION
**Pull request by kevinrizza**
-Update operatorsource spec to include secret ref

### CODE REVIEW

File: pkg/apis/marketplace/v1alpha1/operatorsource_types.go [hunk 1]
```diff
@@ -13,19 +13,6 @@ const (
...
-// OperatorSource is the Schema for the operatorsources API
-// +k8s:openapi-gen=true
-type OperatorSource struct {
-	metav1.TypeMeta   `json:",inline"`
-	metav1.ObjectMeta `json:"metadata,omitempty"`
-	Spec              OperatorSourceSpec   `json:"spec,omitempty"`
-	Status            OperatorSourceStatus `json:"status,omitempty"`
-}
-
 // Only type definitions go into this file.
```
Comment (tkashem) [inline pkg/apis/marketplace/v1alpha1/operatorsource_types.go]: Did you have to move this manually to get it to work?

Comment (kevinrizza) [reply pkg/apis/marketplace/v1alpha1/operatorsource_types.go]: Not as far as I know. I didn't rerun the sdk to generate these, just copied them from your working branch. Would you prefer that we rerun the sdk again to generate this?

**Comment by kevinrizza** [comment]
no longer necessary as a separate pr

**Pull request closed without merging**